### ISSUE TITLE
Indentation not fixed completely when adding and removing curly brace

### ISSUE THREAD

**Issue by finn-matti** [wongjn/vscode-php-sniffer#36]
**Describe the bug**
When using format on save ad when accidently adding a curly brace (or similar), saving, removing the brace and saving again, the indentation is not fixed completely.

Gif showing the problem:

![Kapture 2020-03-25 at 15 35 01](https://user-images.githubusercontent.com/56148301/77542273-9c11d280-6eae-11ea-8834-eed7529c5fe6.gif)

**To Reproduce**
1. Add curly brace, where it doesn't belong
2. Save
3. Remove curly brace
4. Save again

**Expected behavior**
Everything should look like before with all correct indentations.

**Environment**
 - OS: macOS 10.15.4
 - VSCode version: 1.43.2
 - PHP CLI version: 7.3.11
 - squizlabs/PHP_CodeSniffer version: 3.4.2 (global ✔)

**Extension settings**
```json
{
    "phpSniffer.run": "onSave",
    "phpSniffer.onTypeDelay": 250,
    "phpSniffer.executablesFolder": "",
    "phpSniffer.autoDetect": false,
    "phpSniffer.standard": "~/phpcs.xml",
    "phpSniffer.snippetExcludeSniffs" : [],
}
```

**Additional context**
Contents of phpcs.xml

```
<?xml version="1.0" ?>
<ruleset name="Custom Standard">
    <rule ref="PSR2">
        <exclude name="PSR2.ControlStructures.ControlStructureSpacing.SpacingAfterOpenBrace" />
    </rule>
</ruleset>
```

**Comment by wongjn** [comment]
This is something to do with PHP_CodeSniffer package itself, not this extension.

You can see this by running `phpcbf` from the command line, e.g.

Given `./index.php` that contains:
```php
<?php

<?php
$a = [
'b' => [
'c' => 1,
]
];
```

Running `./vendor/bin/phpcbf --standard=PSR2 ./index.php` does nothing.

Please consider raising an issue in the squizlabs/PHP_CodeSniffer repo.
